sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 - Removed unnecessary dependencies on non-Windows platforms. (A community contribution, courtesy of _[teo-tsirpanis](https://github.com/teo-tsirpanis)_)
+- Added support for 'azure-data-tables' connection strings. 
```

Comment:
You no longer list `azure-storage-common` as a dependency, so if this is right, then you won't be using this functionality, and should move the parser somewhere else (to `azure-data-tables-cpp`?), and then your PR would have zero changes for `sdk/storage`.

Otherwise, you need to list `azure-storage-common-cpp` as a dependency, but then I am not sure if `azure-core-xml-cpp` has to exist at all, so it is probably now what you wanted.